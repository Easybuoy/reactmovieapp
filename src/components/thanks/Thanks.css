body{
    background: linear-gradient(89deg, #FF5EDF 0%, #04C8DE 100%); 
}

h1, h3{
    color: white;
}

.btn-outline-brown{
    color: white !important;
}