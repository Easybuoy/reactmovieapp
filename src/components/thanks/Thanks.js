import React, { Component } from 'react';

class Thanks extends Component {

  render(){

 
    return(
      <div className="container con">
      
      <div style={{marginTop: '10%', marginBottom: '10%'}}>
      <h1>Thanks For Reaching Out</h1>
      <h3> I Will Be In Touch Shortly</h3>
      <a href='/'><input className="btn btn-outline-brown" type="button" value="Back To Home Page " /></a>
      </div>


      </div>
    );
  }
}

export default Thanks;

