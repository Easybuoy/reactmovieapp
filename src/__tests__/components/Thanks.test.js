/* eslint-disable no-undef */
import React from 'react';
import { shallow } from 'enzyme';
import Thanks from '../../components/thanks/Thanks';

describe('<Thanks />', () => {

  it('renders the Thanks component correctly', () => {
    const wrapper = shallow(<Thanks />);
    expect(wrapper).toMatchSnapshot();
});
})

