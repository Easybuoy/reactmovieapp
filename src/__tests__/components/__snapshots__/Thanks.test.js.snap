// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Thanks /> renders the Thanks component correctly 1`] = `ShallowWrapper {}`;
